docs/bundle-diffs.md

```diff
@@ -0,0 +1,265 @@
+            }
+          }
+        ],
+        "tolerations": []
+      }
+    }
+  }
+}
+```
+
+In this case there is only 1 container in the deployments container spec, and that container has cpu limits and tolerations added.
+
+Based on this information our diff patch would look as follows:
+```
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: gatekeeper-controller-manager
+    namespace: cattle-gatekeeper-system
+    operations:
+    - {"op": "remove", "path": "/spec/template/spec/containers/0/resources/limits/cpu"}
+    - {"op": "remove", "path": "/spec/template/spec/tolerations"}
+```
+
+### 3. Deployment gatekeeper-audit:
+The gatekeeper-audit deployment is modified in a similar manner to the gatekeeper-controller-manager, with additional cpu limits and tolerations applied.
+
+```
+{
+  "spec": {
+    "template": {
+      "spec": {
+        "$setElementOrder/containers": [
+          {
+            "name": "manager"
+          }
+        ],
+        "containers": [
+          {
+            "name": "manager",
+            "resources": {
+              "limits": {
+                "cpu": "1000m"
+              }
+            }
+          }
+        ],
+        "tolerations": []
+      }
+    }
+  }
+}
+```
+
+Similar to gatekeeper-controller-manager there is only 1 container in the deployments container spec, and that has cpu limits and tolerations added.
+
+Based on this information our diff patch would look as follows:
+```
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: gatekeeper-audit
+    namespace: cattle-gatekeeper-system
+    operations:
+    - {"op": "remove", "path": "/spec/template/spec/containers/0/resources/limits/cpu"}
+    - {"op": "remove", "path": "/spec/template/spec/tolerations"}
+```
+
+### Combining it all together
+We can now combine all these patches as follows:
+
+```
+diff:
+  comparePatches:
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: gatekeeper-audit
+    namespace: cattle-gatekeeper-system
+    operations:
+    - {"op": "remove", "path": "/spec/template/spec/containers/0/resources/limits/cpu"}
+    - {"op": "remove", "path": "/spec/template/spec/tolerations"}
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: gatekeeper-controller-manager
+    namespace: cattle-gatekeeper-system
+    operations:
+    - {"op": "remove", "path": "/spec/template/spec/containers/0/resources/limits/cpu"}
+    - {"op": "remove", "path": "/spec/template/spec/tolerations"}
+  - apiVersion: admissionregistration.k8s.io/v1beta1
+    kind: ValidatingWebhookConfiguration
+    name: gatekeeper-validating-webhook-configuration
+    operations:
+    - {"op": "remove", "path":"/webhooks/0/clientConfig/caBundle"}
+    - {"op": "remove", "path":"/webhooks/0/rules"}
+    - {"op": "remove", "path":"/webhooks/1/clientConfig/caBundle"}
+    - {"op": "remove", "path":"/webhooks/1/rules"}
+```
+
+We can add these now to the bundle directly to test and also commit the same to the fleet.yaml in your GitRepo.
```

Comment:
nit: Wrap fleet.yaml with \` since the "." might be interpreted as a sentence ending.